example_files/Gemfile

```diff
@@ -66,13 +66,14 @@ else
   gem 'openstudio-calibration', '~> 0.5.0'
 end
 
-if allow_local && File.exists?('../urbanopt-geojson-gem')
-  gem 'urbanopt-geojson', path: '../urbanopt-geojson-gem'
-elsif allow_local
-  gem 'urbanopt-geojson', github: 'URBANopt/urbanopt-geojson-gem', branch: 'develop'
-else
-  gem 'urbanopt-geojson', '~> 0.7.0'
-end
+# Temporarily pointing to GeoJSON Gem Branch
+#if allow_local && File.exists?('../urbanopt-geojson-gem')
+#  gem 'urbanopt-geojson', path: '../urbanopt-geojson-gem'
+#elsif allow_local
+  gem 'urbanopt-geojson', github: 'URBANopt/urbanopt-geojson-gem', branch: 'res_enhancement'
```

Comment:
Do we want this to be merged with the branch pointing to `res_enhancement`?

Reply:
@joseph-robertson I have requested your review on the geojson gem, we could change this to develop and eventually to the released geojson gem when we merge this PR.